Build

livebylife opened the pull request.

livebylife committed:
More clean up and optimization. Lazy loaded messenger chat

livebylife committed:
Tuesdays build

ghost commented:
# [Gatsby Cloud](https://www.gatsbyjs.com) Build Report
**LivebylifeTechnologies-main**

:tada: Your build was successful! [See the Deploy preview here.](https://livebylifetechnologiesmain-build.gtsb.io)

## Build Details

[View the build logs here.](https://gatsbyjs.com/dashboard/fc71b8a4-f1c7-4b4a-bfd5-c740fb01084f/sites/492e1238-e90b-48bb-982b-78a2bd3847d8/builds/d60d89fa-d8de-4b80-ad57-207d3ce420bb/details)

:clock1: Build time: **2m**

## Performance

### Lighthouse report

| Metric | Score |
| ----- | ---- |
| Performance |   :large_orange_diamond: 89   |
| Accessibility |   :green_heart: 100   |
| Best Practices |   :green_heart: 92   |
| SEO |   :green_heart: 92   |

:link: [View full report](https://build-d60d89fa-d8de-4b80-ad57-207d3ce420bb.gtsb.io/reports/lighthouse/index.html)

livebylife committed:
added cpanel yml file to gitignore

livebylife commented:
Looks good

livebylife committed:
Logged some comments

Troubleshooting why contact form is not posting through to the API when hosted on Gatsby Cloud

livebylife committed:
Updated contact form

form now includes action and method.

livebylife committed:
trying something

livebylife committed:
Continued troubleshooting contact form post problem

livebylife committed:
July 28 2022 Build

livebylife committed:
Updated contact component

livebylife commented:
Let's keep improving the performance score from lighthouse
